client/project_policy.go

```diff
@@ -19,8 +19,8 @@ type Policy struct {
 
 type PolicyUpdatePayload struct {
 	ProjectId                   string `json:"projectId"`
-	NumberOfEnvironments        *int   `json:"numberOfEnvironments,omitempty"`
-	NumberOfEnvironmentsTotal   *int   `json:"numberOfEnvironmentsTotal,omitempty"`
+	NumberOfEnvironments        *int   `json:"numberOfEnvironments" tfschema:",omitempty"`
```

Comment:
Removed the omitempty for the json.
In general it shouldn't be required, but it seems to be an issue with how the backend works (expects "null").